uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,64 @@
+var Promise = require('es6-promise').Promise;
+var page = require('./page');
+
+module.exports = (function () {
+
+  return page.extend({
+
+    /**
+     * [openMyReport description]
+     * @return {[Page]} [description]
+     */
+    openMyReport: function () {
+      var casper = this.casper;
+
+      casper.then(function () {
+        casper.waitUntilVisible('td[ng-click="report.toggleSection(\'pending\')"]', function () {
+          casper.wait(700);
+        });
+      });
+
+      return this;
+    },
+
+    /**
+     * [openMyReportSection description]
+     */
+    openMyReportSection: function (section) {
+      var casper = this.casper;
+
+      casper.then(function () {
+        casper.click('td[ng-click="report.toggleSection(' + section + ')"]');
+      });
+    },
+
+    /**
+     * [openActionsMyReport description]
+     */
+    openActionsMyReport: function () {
+      var casper = this.casper;
+
+      casper.then(function () {
+        casper.waitForSelector('tr:nth-child(1)  div[uib-dropdown] a:nth-child(1)', function () {
+          casper.click('tr:nth-child(1)  div[uib-dropdown] a:nth-child(1)');
+        });
+      });
+    },
+
+    /**
+     * [editRequestForMyReport description]
```

Comment:
Dont use [], if you have done this by following other files. Whose are wrong and already updated in 1.6. Please follow standard documentation rules. Same for others.